### PR TITLE
Update angular up to 13 version

### PR DESCRIPTION
**Pull request by Sumragen**

### CODE REVIEW

File: proxy.conf.js [hunk 1]
```diff
@@ -7,7 +7,8 @@ const PROXY_CONFIG = [
         pathRewrite: {
             [PROXY_SERVER_URL_REG]: '',
         },
-        target: "http://xm.test.xm-online.com.ua",
+        // target: "http://xm.test.xm-online.com.ua",
+        target: 'http://172.20.75.176/',
```
Comment (Kaskyi) [inline proxy.conf.js:11]: remove

File: packages/shared/operators/src/download.ts [hunk 2]
```diff
@@ -1,6 +1,8 @@
 export function download(data: string, filename: string, type: string): void {
     const file = new Blob([data], {type});
+    // @ts-ignore
```
Comment (Kaskyi) [inline packages/shared/operators/src/download.ts:3]: use types

File: packages/cli/src/ext-themes-command.ts [hunk 3]
```diff
@@ -26,21 +26,16 @@ export class ExtThemesCommand implements Command {
             const name = matches[1];
             const outFile = `${this.destPath}/${name}.css`;
 
-            sass.render({
-                file,
-                includePaths: ['src', 'src/styles', this.config.extDir],
-                importer: packageImporter(),
+            const res = sass.compile(file, {
+                // includePaths: ['src', 'src/styles', this.config.extDir],
```
Comment (Kaskyi) [inline packages/cli/src/ext-themes-command.ts:30]: remove

File: packages/components/sidebar-right/xm-sidebar-right.module.ts [hunk 4]
```diff
@@ -71,7 +71,10 @@ export class XmSidebarRight implements OnInit, OnDestroy {
         const viewContainerRef = this.xmContainerOutlet.viewContainerRef;
         const componentFactoryResolver = config.injector.get(ComponentFactoryResolver);
         const cfr = componentFactoryResolver.resolveComponentFactory(templateRef);
-        const c = viewContainerRef.createComponent<T>(cfr, null, config.injector);
+        const c = viewContainerRef.createComponent<T>(cfr.componentType, {
+            injector: config.injector,
+            ngModuleRef: cfr['ngModule']
```
Comment (Kaskyi) [inline packages/components/sidebar-right/xm-sidebar-right.module.ts:76]: refactor

**Review by Kaskyi — CHANGES_REQUESTED** [review]

File: packages/components/sidebar-right/xm-sidebar-right.module.ts (hunk 4 above)
Comment (Kaskyi) [reply packages/components/sidebar-right/xm-sidebar-right.module.ts:76]: and squash

File: package.json [hunk 5]
```diff
@@ -50,28 +50,28 @@
     "import-jsf-exts": "node ./scripts/jsf-ext-importer.js",
     "translations": "npm run get-translations && xm-ngx ext-i18n",
     "report:bundle": "webpack-bundle-analyzer dist/stats.json",
-    "get-version": "printf $npm_package_version",
-    "postinstall": "node ./scripts/build-packages.js && ngcc --properties esm5 browser module main"
+    "postinstall": "node ./scripts/build-packages.js && ngcc --properties esm5 browser module main",
+    "get-version": "printf $npm_package_version"
```
Comment (major13ua) [inline package.json:54]: Please update major version of application

File: package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "xm-webapp",
-  "version": "3.6.98",
+  "version": "3.6.99",
```
Comment (sergeysenja1992) [inline package.json:3]: update major version

File: packages/cli/src/ext-themes-command.ts [hunk 7]
```diff
@@ -5,7 +5,6 @@ import * as sass from 'sass';
 import * as path from 'path';
 import { Command } from './command';
 import { Config } from './config';
-import { packageImporter } from './ext-theming/node-sass-package-importer';
```
Comment (Kaskyi) [inline packages/cli/src/ext-themes-command.ts:8]: looks like we no need `packageImporter`.
Please , remove all files from `packages/cli/src/ext-theming`

File: packages/components/sidebar-right/xm-sidebar-right.module.ts (hunk 4 above)
Comment (Kaskyi) [inline packages/components/sidebar-right/xm-sidebar-right.module.ts:76]: there must be a public variable somewhere.

**Review by Kaskyi — CHANGES_REQUESTED** [review]

**Review by sergeysenja1992 — APPROVED** [review]

**Pull request closed without merging**